core/domain/exp_jobs_one_off.py

```diff
@@ -367,13 +367,23 @@ def map(item):
             return
 
         # Do not upgrade explorations that fail non-strict validation.
-        old_exploration = exp_fetchers.get_exploration_by_id(item.id)
+        try:
+            old_exploration = exp_fetchers.get_exploration_by_id(item.id)
+        except Exception as e:
+            error_message = (
+                'Exploration %s failed migration during GET: %s' %
+                (item.id, e))
+            logging.error(error_message)
```

Comment:
Should this be logging.exception? Ditto below/elsewhere.

Reply:
Done